OpenShift-Deployment-Guide.md

```diff
@@ -0,0 +1,54 @@
+I was so happy. Took me more than 10 days to finish my voting app, my first web app project at Free Code Camp. I was ready to deploy to heroku and...
```

Comment:
I would recommend that you make it less personal. People will be reading this as if it was a Free Code Camp's article and FCC is not a person.